frontend/src/Component/Banner/BannerSlider.tsx

```diff
@@ -0,0 +1,30 @@
+import { ReactNode } from 'react';
+import { Swiper, SwiperSlide } from 'swiper/react';
+import SwiperCore, { Navigation, Scrollbar, Autoplay } from 'swiper';
+
+import 'swiper/swiper-bundle.min.css';
+import 'swiper/swiper.min.css';
+
+interface BannerProps {
+  children: ReactNode;
+  _banner: string[];
+}
+
+const BannerSlider = ({ children, _banner }: BannerProps) => {
```

Comment:
```suggestion
const BannerSlider: React.FC<BannerProps> = ({ children, _banner }) => {
```

리액트 컴포넌트에 Props타입을 줄 때는 아래처럼 사용하는 방식을 많이 선호합니다!

Reply:
+ 변수명 앞에 `_` 접두어를 쓴 이유가 있나요?

Reply:
습관적으로 _를 붙여버렸네요 ㅎㅎ 수정하겠습니다!